### PR TITLE
docs describe how to remove healthy Ceph OSDs

### PR DESCRIPTION
**Pull request by satoru-takeuchi**
**Description of your changes:**

Current document of OSD removal only describe the case of broken OSDs.

**Which issue is resolved by this Pull Request:**

Nothing

**Checklist:**

- [o] **Commit Message Formatting**: Commit titles and messages follow guidelines in the [developer guide](https://rook.io/docs/rook/master/development-flow.html#commit-structure).
- [o] **Skip Tests for Docs**: Add the flag for skipping the build if this is only a documentation change. See [here](https://github.com/rook/rook/blob/master/INSTALL.md#skip-ci) for the flag.
- [o] **Skip Unrelated Tests**: Add a flag to run tests for a specific storage provider. See [test options](https://github.com/rook/rook/blob/master/INSTALL.md#test-storage-provider).
- [o] Reviewed the developer guide on [Submitting a Pull Request](https://rook.io/docs/rook/master/development-flow.html#submitting-a-pull-request)
- [o] Documentation has been updated, if necessary.
- [o] Unit tests have been added, if necessary.
- [o] Integration tests have been added, if necessary.
- [o] Pending release notes updated with breaking and/or notable changes, if necessary.
- [o] Upgrade from previous release is tested and upgrade user guide is updated, if necessary.
- [o] Code generation (`make codegen`) has been run to update object specifications, if necessary.

[skip ci]

### CODE REVIEW

File: Documentation/ceph-osd-mgmt.md [hunk 1]
```diff
@@ -66,7 +66,7 @@ and it is safe to proceed. If an OSD is failing, the PGs will not be perfectly c
 ### From the Toolbox
 
 1. Determine the OSD ID for the OSD to be removed. The osd pod may be in an error state such as `CrashLoopBackoff` or the `ceph` commands
-in the toolbox may show which OSD is `down`.
+in the toolbox may show which OSD is `down`. If you want to remove a healthy OSD, you set the corresponding deployment replicas to 0 beforehand.
```
Comment (leseb) [inline Documentation/ceph-osd-mgmt.md:69]: Can you add the command for the user to do so? So the doc has everything

Comment (satoru-takeuchi) [reply Documentation/ceph-osd-mgmt.md]: Added, thanks.

File: Documentation/ceph-osd-mgmt.md [hunk 2]
```diff
@@ -66,7 +66,7 @@ and it is safe to proceed. If an OSD is failing, the PGs will not be perfectly c
 ### From the Toolbox
 
 1. Determine the OSD ID for the OSD to be removed. The osd pod may be in an error state such as `CrashLoopBackoff` or the `ceph` commands
-in the toolbox may show which OSD is `down`.
+in the toolbox may show which OSD is `down`. If you want to remove a healthy OSD, you should run `kubectl -n <NS> scale deployment rook-ceph-deployment-<ID> --replicas=0` beforehand.
```
Comment (leseb) [inline Documentation/ceph-osd-mgmt.md:69]:
```suggestion
in the toolbox may show which OSD is `down`. If you want to remove a healthy OSD, you should run `kubectl -n rook-ceph scale deployment rook-ceph-osd-<ID> --replicas=0` beforehand.
```

The entire doc assumes rook-ceph so it's fine to keep it, also I think you meant `rook-ceph-osd`?

**Review by leseb — CHANGES_REQUESTED** [review]

File: Documentation/ceph-osd-mgmt.md (hunk 2 above)
Comment (satoru-takeuchi) [reply Documentation/ceph-osd-mgmt.md]: Indeed, fixed.

**Review by leseb — APPROVED** [review]

**Review by travisn — APPROVED** [review]

**Pull request merged**